### ISSUE TITLE
Deploy dev branch to staging action failing

### ISSUE THREAD

**Issue by anjor** [application-research/estuary#665]
**Describe the bug**
https://github.com/application-research/estuary/actions/workflows/deploy-staging.yml has been failing for the last 17 days.

**Comment by LucRoy** [comment]
I think they are still using staging for testing AR if im not mistaken. @jlogelin @alvin-reyes @elijaharita 

The issues is because we had a manual instance of estuary deployed to staging for testing AR. The pipeline is broken since the port is already in use. If we are done with testing, can we clean up staging?

**Comment by elijaharita** [comment]
> I think they are still using staging for testing AR if im not mistaken. @jlogelin @alvin-reyes @elijaharita 
> 
> The issues is because we had a manual instance of estuary deployed to staging for testing AR. The pipeline is broken since the port is already in use. If we are done with testing, can we clean up staging?

right, this was the issue. i will shut down the manual instance to allow deploy script to run
